ui/mods/mod_reforged/js_hooks/screens/character/modules/character_screen_left_panel/character_screen_stats_module.js

```diff
@@ -15,12 +15,12 @@ CharacterScreenStatsModule.prototype.setProgressbarValues = function (_data)
 	{
 		if (_value.TooltipId == "character-stats.Morale")
 		{
-			_value.Row.bindTooltip({ contentType: 'msu-generic', modId: Reforged.ID, elementId: "Concept.Reach" });
+			_value.Row.bindTooltip({ entityId: entityID, contentType: 'msu-generic', elementId: "Concept.Reach", modId: Reforged.ID });
```

Comment:
Just move this around to to have the same order:
`{ contentType: 'msu-generic', modId: Reforged.ID, elementId: "Concept.Reach",  entityId: entityID}`

Reply:
But the `else` directly below it also has `entityId: entityID` as the first argument. Should I change that too?

Reply:
I think usually it's type, modid, elementid, extra content, so yeah, you can change those too. Thanks